.golangci.yaml

```diff
@@ -3,6 +3,7 @@ run:
   skip-dirs:
     - internal/plugin
     - extensions/functions/tfLite
+    - internal/pkg/httpx/httptestx
```

Comment:
why skipping this dir?

Reply:
cuz the codes under this package uses package with prefix other than `github.com/lf-edge/ekuiper`, the GCI tests cannot pass if these are not ignored.